Model/ModelManager.php

```diff
@@ -509,7 +509,12 @@ public function getDataSourceIterator(DatagridInterface $datagrid, array $fields
         $query->setMaxResults($maxResult);
 
         if ($query instanceof ProxyQueryInterface) {
-            $query->addOrderBy($query->getSortBy(), $query->getSortOrder());
+            $sortBy = $query->getSortBy();
+            $sortOrder = $query->getSortOrder();
+
+            if (!empty($sortBy) && !empty($sortOrder)) {
+                $query->addOrderBy($query->getSortBy(), $query->getSortOrder());
```

Comment:
Why calling again `$query->getSortBy()` and `$query->getSortOrder()` here? You can just use the variables.